pkg/frameworks/tensorflow/tensorflow_serving.go

```diff
@@ -28,16 +28,24 @@ const (
 	TensorflowServingImageName  = "tensorflow/serving"
 )
 
-func CreateTensorflowContainer(modelName string, tfSpec *v1alpha1.TensorflowSpec) *v1.Container {
+type TensorflowFramework struct {
+	Spec *v1alpha1.TensorflowSpec
```

Comment:
Same as above, we can implement the interface directly on `TensorflowSpec`

Reply:
+1 I was just about to suggest this!

Reply:
used a type alias - cant use the same type since it is defined in a different package